stacks/java-wildfly/devfile.yaml

```diff
@@ -96,142 +30,95 @@ components:
       env:
         - name: GC_METASPACE_SIZE
           value: '96'
-        - name: GC_MAX_METASPACE_SIZE
-          value: '256'
-        - name: GALLEON_PROVISION_LAYERS
-          value: 'jaxrs-server,microprofile-platform'
-        - name: GALLEON_LOCAL_MAVEN_REPO
-          value: /tmp/galleon-m2-repository
-        - name: S2I_DESTINATION_DIR
-          value: '/projects'
-        - name: S2I_DELETE_SOURCE
-          value: 'false'
-        - name: STANDALONE_RESTORE
-          value: /home/jboss/.m2/.lucky
-        - name: JBOSS_HOME
-          value: '/projects/wildfly'
+        - name: JAVA_OPTS
+          value: '-Djava.security.egd=file:/dev/urandom'
         - name: MVN_ARGS_APPEND
           value: '-s /home/jboss/.m2/settings.xml -Dmaven.repo.local=/home/jboss/.m2/repository -Dcom.redhat.xpaas.repo.jbossorg'
         - name: DEBUG_PORT
-          value: '5858'
+          value: "5005"
       endpoints:
-        - name: http-wildfly
+        - name: debug
+          exposure: internal
+          protocol: tcp
+          targetPort: 5005
+        - name: 'http'
+          protocol: http
           targetPort: 8080
-        - exposure: none
-          name: debug
-          targetPort: 5858
+          exposure: public
+        - name: 'management'
+          targetPort: 9990
+          protocol: http
+          exposure: internal
+        - name: 'transaction'
+          protocol: tcp
+          targetPort: 4172
+          exposure: internal
+  - name: image-build
+    image:
+      imageName: numberguess:latest
+      dockerfile:
+        uri: Containerfile
+        buildContext: .
+        rootRequired: false
+  - name: kubernetes-deploy
+    attributes:
+      deployment/replicas: 1
+      deployment/cpuLimit: '100m'
+      deployment/cpuRequest: 10m
+      deployment/memoryLimit: 300Mi
+      deployment/memoryRequest: 180Mi
+      deployment/container-port: 8081
+    kubernetes:
+      uri: deploy.yaml
```

Comment:
The tests will fail if it can't find the `deploy.yaml` in the stack directory. Would it be possible to add one?

Reply:
I added one

Reply:
The new stack seems to be failing the non terminating image check

```
MountVolume.SetUp failed for volume "kube-api-access-vg96f" : failed to sync configmap cache: timed out waiting for the condition
```